public/gallery.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+	<script src="https://cdn.tailwindcss.com"></script>
```

Comment:
Why is this so up here?

Reply:
i put it there so is the same as the home.html... so is more consistent across the pages

Reply:
I missed that on the home it seems. It doesn't need to be so high. It could easily be at the beginning of the body